src/pages/public_registration/LogPanel.tsx

```diff
@@ -1,16 +1,19 @@
-import { PublishingTicket, Ticket, TicketType } from '../../types/publication_types/ticket.types';
-import { Box, Skeleton, Typography } from '@mui/material';
-import { useTranslation } from 'react-i18next';
+import { Box, Skeleton, Tooltip, Typography } from '@mui/material';
 import { useQuery } from '@tanstack/react-query';
+import { TFunction } from 'i18next';
+import { useTranslation } from 'react-i18next';
 import { fetchOrganization } from '../../api/cristinApi';
 import { fetchUser } from '../../api/roleApi';
+import { PublishingTicket, Ticket, TicketType } from '../../types/publication_types/ticket.types';
 import { Registration } from '../../types/registration.types';
+import { getAssociatedFiles } from '../../utils/registration-helpers';
 import { getFullName } from '../../utils/user-helpers';
 import { StyledStatusMessageBox } from '../messages/components/PublishingRequestMessagesColumn';
 
 interface LogItem {
   modifiedDate: string;
   description: string;
+  descriptionAppendage?: string[];
```

Comment:
appendage? 🤔 
https://www.merriam-webster.com/dictionary/appendage
https://en.wikipedia.org/wiki/Appendage

Reply:
i dette tilfellet ser det bare brukes for filnavn, så kanskje `fileNames` er like greit?

Reply:
Så bare på det første punktet i merriam-webster 😅 Tanken var å lage log-interfacet generelt, for framtidig bruk, men kan heller gjøre det generelt når det er bruk for det 👍